model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -107,6 +107,12 @@ class LicenseInfoResolver(
             license.builder().apply {
                 sources += LicenseSource.DETECTED
                 resolvedLocations[license]?.let { locations.addAll(it) }
+
```

Comment:
Commit message nits:
- Please prefix the title with "LicenseInfoResolver: "
- s/Enhance test/Enhance the test/
- Maybe: s/made it unclear/is unclear about/